.gitignore

```diff
@@ -1,4 +1,7 @@
 /docs/doxygen/*
+# the GitHub Actions documentation job
+# needs to commit the html output
+!/docs/doxygen/html
```

Comment:
I think the only reason to have the gitignore for `/docs/doxygen/*` above was to exclude the generated doxygen documentation from being pushed. If we now excplicitly include the generated html files we could also directly remove the exclusion.

Reply:
Yes, it was the idea of the exclude. But it also excludes the `xml` output. However, I changed to your approach. At the first time, when I chose a solution, I thought modifying the `.gitignore` is the better solution than using the `--force` argument. But you are right, you approach is better suited for normal developer.